test/tokens/ERC721.test.js

```diff
@@ -40,8 +40,8 @@ contract('ERC721', (accounts) => {
     await assertRevert(nftoken.mint('0', id3));
   });
 
-  it('throws when trying to mint NFToken from non owner ot authorized address', async () => {
-    await assertRevert(nftoken.mint('0', id3, { from: accounts[1] }));
+  it.skip('throws when trying to mint NFToken from non-owner', async () => {
+    await assertRevert(nftoken.mint(accounts[1], id3, { from: accounts[1] }));
```

Comment:
@MoMannn we need to decide what to do with this. Maybe we should just remove it since we don't have any permission modifiers on `_mint`?

Reply:
We should remove it.